### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: ros_opcua_impl_python_opcua/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ros_opcua_impl_python_opcua
+
+This package provides OPC UA server implementation for connection to ROS. Server provides access to all ROS Topics and services from an  OPC UA Client.
```
Comment (yangpeiren) [inline ros_opcua_impl_python_opcua/README.md]: and ros actions

File: ros_opcua_impl_python_opcua/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# ros_opcua_impl_python_opcua
+
+This package provides OPC UA server implementation for connection to ROS. Server provides access to all ROS Topics and services from an  OPC UA Client. 
+
+## Getting started with ros_opcua_impl_python_opcua package
+
+First start some ROS Nodes for something to be visalized in the server. For example one can start ``turtlesim_node`:
```
Comment (yangpeiren) [inline ros_opcua_impl_python_opcua/README.md]: one redundant \`  before `turtlesim_node`

File: ros_opcua_impl_python_opcua/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+```
+rosrun turtlesim turtlesim_node 
+```
+
+Then start the server (the starting will take some time, wait until you see the output) using:
+```
+roslaunch ros_opcua_impl_python_opcua rosopcua.launch
+```
+
+Then start the *UaExpert OPC UA* Client and connect to the server. Server can be accessed per default from wholw local network on port `21554`. If you are running everything localy use `opc.tcp://localhost:21554` as server URL.
```
Comment (yangpeiren) [inline ros_opcua_impl_python_opcua/README.md]: the wholw maybe the

File: ros_opcua_impl_python_opcua/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+After successful connection you can see all ROS Services, Topics and Actions mapped to the OPC UA. To move the turtle from exaple choose `Objects->ROS-Services->turtle1/teleport_absolute` with right clieck and choose call. Enter the new possitionof the turtle and see how turtle moves.
```
Comment (yangpeiren) [inline ros_opcua_impl_python_opcua/README.md]: possitionof forget space

**Review by yangpeiren — COMMENTED** [review]
This is a specification for the current version, correct? Maybe after the new version merged to the main repository we need to update this specification.

**Review by yangpeiren — COMMENTED** [review]
This is a specification for the current version, correct? Maybe after the new version merged to the main repository we need to update this specification.

Sorry for the repetition, I did not find the delete button...

**Comment by destogl** [comment]
Yes, this is the specification for the current version. After merge of your work, we will adapt the description.

**Review by yangpeiren — APPROVED** [review]

**Pull request merged**